pandas/core/generic.py

```diff
@@ -4818,7 +4818,8 @@ def sample(
         frac : float, optional
             Fraction of axis items to return. Cannot be used with `n`.
         replace : bool, default False
-            Sample with or without replacement.
+            Sample with or without replacement. Setting this to `True` means that a
+            record in the original data can appear multiple times in the sampled data.
```

Comment:
Can you add an example to the Examples section of the docstring where this is used in a meaningful way? Specifically where rows are resampled, as the current example with `replace=True` doesn't actually show this behavior.